### ISSUE TITLE
Help Slot Not Displayed Unless Help Property Exists

### ISSUE THREAD

**Issue by TheBroox** [JosephusPaye/Keen-UI#301]
The help slot on the textbox element will not display unless the help property is set to something in the textbox opening tag. This seems like undesirable behavior and is at the very least unintuitive since no other slots that I am aware of behave like this. The examples below illustrate what I mean.

Examples 1: Slot Won't Display
```
<ui-textbox floating-label v-model="password" type="password">
    Password
    <template slot="help">
        <div>This text won't display</div>
    </template>
</ui-textbox>
```

Example 2: Slot Will Display But Overwrite Property
```
<ui-textbox floating-label v-model="password" type="password" help="This text wont display">
    Password
    <template slot="help">
        <div>This text will display</div>
    </template>
</ui-textbox>
```

Example 3: Noting Need To Be Passed to the Property
```
<ui-textbox floating-label v-model="password" type="password" help>
    Password
    <template slot="help">
        <div>This text will display</div>
    </template>
</ui-textbox>
```

**Comment by JosephusPaye** [comment]
Hi @TheBroox,

Example 1 and 3 is a bug, the `help` prop and slot should be usable independently. This will be fixed in the next release.

Example 2 is expected behavior, if both the prop and the slot are set, the slot overrides the prop.

**Comment by JosephusPaye** [comment]
Fixed in https://github.com/JosephusPaye/Keen-UI/commit/63d54a6ddcaa6f1f81d604cda0e856b9a5512897.

**Comment by JosephusPaye** [comment]
Fix released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
